### PR TITLE
Update Kapotun data

### PR DESCRIPTION
**Pull request by IllusiveMan196**

### CODE REVIEW

File: masterlist.yaml [hunk 1]
```diff
@@ -9934,17 +9937,23 @@ plugins:
...
           - 'Touched by Dibella'
-          - '[Races of Nirn - Kapotun Reborn](https://www.nexusmods.com/skyrimspecialedition/mods/77612/)'
+          - '[Fixed Races of Nirn - Kapo''tun SE port](https://www.nexusmods.com/skyrimspecialedition/mods/66376/)'
         condition: 'active("TouchedbyDibella.esp") and checksum("DWKaPoTunRace.esp", 672EB2D2) and not active("Kapotun TBD Tail.esp")'
       - <<: *patchIncluded
         subs: [ 'Schlongs of Skyrim' ]
         condition: 'active("Schlongs of Skyrim - Core.esm") and version("DWKaPoTunRace.esp", "2.0", >=) and not active("SOS - Kapotun Addon.esp")'
+      - <<: *alsoUseX
+        subs: [ '[Fixed Races of Nirn - Kapo''tun SE port](https://www.nexusmods.com/skyrimspecialedition/mods/66376/)' ]
+        condition: 'checksum("DWKaPoTunRace.esp", 672EB2D2)'
```
Comment (pStyl3) [inline masterlist.yaml]: Please sort the `alsoUseX` message alphabetically before the `obsolete` message.

**Review by pStyl3 — CHANGES_REQUESTED** [review]

File: masterlist.yaml (hunk 1 above)
Comment (IllusiveMan196) [reply masterlist.yaml]: Done

**Review by pStyl3 — APPROVED** [review]

File: masterlist.yaml [hunk 2]
```diff
@@ -9922,19 +9922,25 @@ plugins:
         name: 'Races of Nirn - Kapo''tun SE'
       - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/77612/'
         name: 'Races of Nirn - Kapotun Reborn'
+    group: *racesGroup
     inc:
+      - name: 'DWKaPoTunRace.esp'
+        condition: 'checksum("DWKaPoTunRace.esp", D34873CA)'
```
Comment (pStyl3) [inline masterlist.yaml]: What do you want to convey with this `inc` entry? As it is, you are saying that `DWKaPoTunRace.esp` is incompatible with itself (if the checksum is identical). Do you want to convey, that the plugin with that checksum is obsolete?

Comment (IllusiveMan196) [reply masterlist.yaml]: Rather that one version shouldn't be changed to another mid-game. Reborn can't be replaced by original and vice versa, unless upgrade patch is used. May have been wrong message or format chosen.

Comment (IllusiveMan196) [reply masterlist.yaml]: Update: deleted these lines.

**Comment by pStyl3** [comment]
After some more [discussion](https://discord.com/channels/473542112974077963/496196499890241546/1142518549416517762) on Discord, I've merged the PR.
Thanks!

image
image
image

**Pull request merged**